FileManager/FileManager/ViewController.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ViewController.swift
+//  FileManager
+//
+//  Created by Taraf Bin suhaim on 22/04/1443 AH.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    var ArrayFolders = [String]()
+    var folderSelect = ""
+    var folderAlow = ""
+    
+    @IBOutlet weak var textName: UITextField!
+    @IBOutlet weak var textArea: UITextView!
+    @IBOutlet weak var switchOutlet: UISwitch!
+    @IBOutlet weak var tabelView: UITableView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tabelView.delegate = self
+        tabelView.dataSource = self
+        switchOutlet.isOn = false
+        folderAlow = "NotAllow"
+        loadData()
+    }
+    
+    @IBAction func craeteFolder(_ sender: Any) {
+        creatDir(dieName: textName.text!)
+    }
+    
+    @IBAction func craeteFile(_ sender: Any) {
+        createFile(dirName: folderSelect, fileName: textName.text!)
+    }
+    
+    @IBAction func move(_ sender: Any) {
+        if switchOutlet.isOn {
+            folderAlow = "Allow"
+        }else{
+            folderAlow = "NotAllow"
+        }
+    }
+    
+    func loadData(){
+        let fileManger = FileManager.default
+        let dirURL = fileManger.urls(for: .documentDirectory, in: .userDomainMask).first
+        do {
+            let folders = try fileManger.contentsOfDirectory(atPath: dirURL!.path)
+            print(folders)
+            
+            for folder in folders {
+                if dirURL?.appendingPathComponent(folder).hasDirectoryPath == true {
+                  self.ArrayFolders.append(folder)
+                  print(folder)
+                }
+            }
+            self.tabelView.reloadData()
+        }catch {}
```

Comment:
Preferably print the error or message
you can use print(error) to print error that happened